lib/DXIL/DxilShaderFlags.cpp

```diff
@@ -721,20 +721,6 @@ ShaderFlags ShaderFlags::CollectShaderFlags(const Function *F,
         case DXIL::OpCode::TextureGatherRaw:
           hasAdvancedTextureOps = true;
           break;
-        case DXIL::OpCode::WaveMatrix_Add:
-        case DXIL::OpCode::WaveMatrix_Annotate:
-        case DXIL::OpCode::WaveMatrix_Depth:
-        case DXIL::OpCode::WaveMatrix_Fill:
-        case DXIL::OpCode::WaveMatrix_LoadGroupShared:
-        case DXIL::OpCode::WaveMatrix_LoadRawBuf:
-        case DXIL::OpCode::WaveMatrix_Multiply:
-        case DXIL::OpCode::WaveMatrix_MultiplyAccumulate:
-        case DXIL::OpCode::WaveMatrix_ScalarOp:
-        case DXIL::OpCode::WaveMatrix_StoreGroupShared:
-        case DXIL::OpCode::WaveMatrix_StoreRawBuf:
-        case DXIL::OpCode::WaveMatrix_SumAccumulate:
-          hasWaveMMA = true;
```

Comment:
is `hasWaveMMA` still needed?

Reply:
Not sure if we want to remove feature flag about WaveMMA or not.
That part should be the same if WaveMatrix is needed in the future.

Reply:
IMO we're either removing this feature entirely or we're not.  This maybe warrants further discussion outside of the PR?

Reply:
Removed.